Readme.md

```diff
@@ -17,11 +17,14 @@ A react-native wrapper for handling in-app purchases.
 
 ### Add it to your project
 
-1. Make sure you have `rnpm` installed: `npm install rnpm -g`
+1. Install and link the react-native-in-app-utils package:
 
-2. Install with rnpm: `rnpm install react-native-in-app-utils`
+```
+react-native install react-native-in-app-utils
```

Comment:
my thinking here was that rnpm and rn-link both had some pain points. changes of getting it right with manual installation is way more than doing it with rn-link.

Happy to roll with rn-link if you guys think it works fine. also needs rebase.